python/spark_sklearn/test_utils.py

```diff
@@ -23,20 +21,25 @@
 # unittest methods setUpClass() and tearDownClass(), invoked by the nosetest command before
 # and after unit tests are run. This enables us to create one PySpark SparkSession per
 # test fixture. The session can be referred to with self.spark or ClassName.spark.
+#
+# The SparkSession is set up before invoking the class' own set up and torn down after the
+# class' tear down, so you may safer to it in those methods.
```

Comment:
Are you referring to the dropped s for singular possessive ending in s? That's ok.